plugins/katello/2.4/user_guide/content_views/index.md

```diff
@@ -26,7 +26,7 @@ What can a Content View be used for?
 
 ## General Workflow
 
-First [create a product and repository](/plugins/katello/{{ page.version }}/user_guide/content/content.html#creating-a-product) in the library environment and populate the repository with content (by syncing it or uploading content).
+First [create a product and repository](/plugins/katello/{{ page.version }}/user_guide/content/index.html#creating-a-product) in the library environment and populate the repository with content (by syncing it or uploading content).
```

Comment:
This view also has a 'Lifecycle Environments' link that's also broken. Can you fix it too? It has the identical problem of not referring to index.html

Reply:
sure thing!

Reply:
I found more places where broken links were used, I replaced them all.